src/utils.jl

```diff
@@ -104,20 +104,26 @@ function to_vectorSvector(a::AbstractMatrix)
 end
 
 """
-    orthonormal(D, k) -> ws
+    orthonormal([T,] D, k) -> ws
 Return a matrix `ws` with `k` columns, each being
 an `D`-dimensional orthonormal vector.
 
-Always return `SMatrix` for stability reasons.
+Default: returns SMatrix{D, k} if D*k < 100, otherwise Matrix
```

Comment:
From this documentation string it is not clear what `T` should be.

Reply:
Made a comment about that, should be clearer now